src/pages/address/[principal].tsx

```diff
@@ -114,6 +121,7 @@ const SummaryCard = ({ principal, hasTokenBalances, data }: any) => {
 
 interface AddressPageData {
   principal: string;
+  info: AccountDataResponse;
```

Comment:
this could be more descriptive, eg `accountInfo`, or if you are only using the nonce, return just that as `nonce`